test/base/test_batch.py

```diff
@@ -225,6 +227,21 @@ def test_batch_from_to_numpy_without_copy():
     assert c_mem_addr_new == c_mem_addr_orig
 
 
+def test_batch_copy():
+    batch = Batch(a=[3, 4, 5], b=[4, 5, 6])
+    batch2 = Batch({'c': [6, 7, 8], 'b': batch})
+    batch3 = Batch(batch2, copy=True)
+    assert batch2.c is not batch3.c
+    assert batch2.b is not batch3.b
+    assert batch2.b.a is not batch3.b.a
+    assert batch2.b.b is not batch3.b.b
+
+    batch = Batch(a=np.array([Batch(), Batch()], dtype=np.object))
```

Comment:
rather use `[Batch(), Batch()]`. Nobody is going to use `np.array([Batch(), Batch()], dtype=np.object)` explicitly.

Reply:
Fixed in 050f5691e2ea8ce5c7ff0270c0b191334a4eab34 .